linux-tools/rng_tools/rng-tools.sh

```diff
@@ -43,14 +44,10 @@ rng_service=rngd
 #	 
 function tc_local_setup()
 {
+	tc_register "Setting up machine to run the tests"
 	tc_root_or_break || return	
```

Comment:
tc_register will register "Setting up machine to run the tests" as one test and increment TST_TOTAL , but I dont see corresponding tc_pass*/tc_fail* associated to this. If intention is put info in logs, you can use tc_info "Setting up machine to run the tests"